presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -721,48 +785,6 @@ private void revokeRole(String role, String granteeName, PrincipalType granteeTy
         }
     }
 
-    private List<String> getPrestoViews(String databaseName)
```

Comment:
If you move your new method `doGetTablesMatchingParameter` here, the diff will be more readable. In particular, I don't know what have you changed in the method except for adding new params.

place `getTablesMatchingParameter` just above.